src/player.cpp

```diff
@@ -11051,11 +11051,11 @@ void player::do_read( item *book )
             book->typeId() == "cookbook_human" ) {
             fun_bonus = 25;
             add_morale(MORALE_BOOK, fun_bonus, fun_bonus * 3, 60, 30, true, book->type);
-        } if (has_trait("SPIRITUAL") && book->has_flag("INSPIRATIONAL")) {
-              fun_bonus = 15;
+        } else if ( has_trait("SPIRITUAL") && book->has_flag("INSPIRATIONAL") ) {
+            fun_bonus = 15;
```

Comment:
Uh, this is the religious texts; the cannibal cookbook is the next one up.  I'm pretty sure To Serve Man isn't considered Inspirational, so I'm at a loss for why the need to else this if.